frontend/benefit/handler/src/components/applicationReview/deminimisView/DeminimisView.tsx

```diff
@@ -41,7 +41,11 @@ const DeminimisView: React.FC<ApplicationReviewViewProps> = ({ data }) => {
             </$SummaryTableHeader>
           </$GridCell>
           {data.deMinimisAidSet?.map((aid: DeMinimisAid) => (
-            <React.Fragment key={`${aid.granter ?? ''}${aid.grantedAt ?? ''}`}>
+            <React.Fragment
+              key={`${aid.granter ?? ''}${formatDate(
+                new Date(aid.grantedAt || '')
```

Comment:
`{convertToUIDateFormat(aid.grantedAt)}` should do the same? It returns empty string if not valid date or empty value.

Reply:
ok, didnt know that we have the helper, i ll fix